auth/realm/base/src/main/java/org/wildfly/security/auth/realm/KeyStoreBackedSecurityRealm.java

```diff
@@ -211,5 +198,18 @@ public boolean verifyEvidence(final Evidence evidence) throws RealmUnavailableEx
         public boolean exists() throws RealmUnavailableException {
             return getEntry(name) != null;
         }
+        
+        private KeyStore.Entry getEntry(String name) {
+          try {
+              KeyStore.Entry entry = keyStore.getEntry(name, null);
+              if (entry == null) {
+                  log.tracef("KeyStoreRealm: alias [%s] does not exist in KeyStore", name);
+              }
+              return entry;
+          } catch (NoSuchAlgorithmException | UnrecoverableEntryException | KeyStoreException e) {
+              log.tracef(e, "KeyStoreRealm: Obtaining entry [%s] from KeyStore failed", name);
+              return null;
+          }
+      }        
```

Comment:
@albertotn Thank you for the PR! Just a minor, this line and the line 201 has trailing spaces that needs to be removed in order for the style check to pass.

Reply:
I've updated, now running checkstyle:checkstyle it's fine for me